### PR TITLE
[Express Route] IPv6 Global Reach Add/Set-AzExpressRouteCircuitConnection cmdlet changes.

### PR DESCRIPTION
**Pull request by amwate**
## Description
We are in the process of adding support for IPv6 in ExpressRouteCircuitConnectionConfig.

In the process, we have updated the existing PowerShell cmdlet 
Add-AzExpressRouteCircuitConnectionConfig to accept another optional parameter AddressPrefixType (that optionally specifies the address family type for the address prefix). 

In addition, we also introduced another cmdlet to modify the existing ExpressRouteCircuitConnectionConfig to add a new block IPv6CircuitConnectionConfig as well. 
This would be used to update any existing GlobalReach configuration (ExpressRouteCircuitConnection) configurations.

Below is the design PR
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/451#event-2913187208

This pull request brings in the changes in the powershell sdk. 
The relevant swagger changes are to be merged in network-december-release.
https://github.com/Azure/azure-rest-api-specs/pull/8145

NRP changes are already merged. 

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Network/Network/Common/NetworkResourceManagerProfile.cs [hunk 1]
```diff
@@ -796,10 +798,10 @@ private static void Initialize()
                 // Express Route Circuit Connection 
                 // CNM to MNM
                 cfg.CreateMap<CNM.PSExpressRouteCircuitConnection, MNM.ExpressRouteCircuitConnection>();
-
+
```
Comment (msJinLei) [inline src/Network/Network/Common/NetworkResourceManagerProfile.cs:801]: Remove redundant spaces

File: src/Network/Network/Common/NetworkResourceManagerProfile.cs [hunk 2]
```diff
@@ -796,10 +798,10 @@ private static void Initialize()
                 // Express Route Circuit Connection 
                 // CNM to MNM
                 cfg.CreateMap<CNM.PSExpressRouteCircuitConnection, MNM.ExpressRouteCircuitConnection>();
-
+                
                 // MNM to CNM 
                 cfg.CreateMap<MNM.ExpressRouteCircuitConnection, CNM.PSExpressRouteCircuitConnection>();
-
+
```
Comment (msJinLei) [inline src/Network/Network/Common/NetworkResourceManagerProfile.cs:804]: Remove redundant spaces

File: src/Network/Network/ExpressRouteCircuit/Peering/Connection/SetAzureExpressRouteCircuitConnectionConfigCommand.cs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
+using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network {
+    [CmdletOutputBreakingChange(typeof(PSExpressRouteCircuit), DeprecatedOutputProperties = new[] { "AllowGlobalReach" })]
```
Comment (msJinLei) [inline src/Network/Network/ExpressRouteCircuit/Peering/Connection/SetAzureExpressRouteCircuitConnectionConfigCommand.cs:22]: Breaking change notification is released ahead of the breaking change for an existing cmdlet. Here is a brand new one and so please remove it.

File: src/Network/Network/Common/NetworkResourceManagerProfile.cs (hunk 1 above)
Comment (amwate) [reply src/Network/Network/Common/NetworkResourceManagerProfile.cs:801]: Done

File: src/Network/Network/Common/NetworkResourceManagerProfile.cs (hunk 2 above)
Comment (amwate) [reply src/Network/Network/Common/NetworkResourceManagerProfile.cs:804]: Done

**Pull request merged**